nixos/modules/services/networking/avahi-daemon.nix

```diff
@@ -204,4 +204,8 @@ in
 
   };
 
+  meta.tests = {
+    "avahi" = ./tests/avahi.nix;
```

Comment:
This is inconsistent with the rest of the attributes where the test-name is without quotation.